knowrob/rules/src/data_vis/canvas.pl

```diff
@@ -10,6 +10,18 @@
 	result_set_events(ResultSet,Evts),
 	openease_canvas(QueryID,Evts).
 
+%%
+% Generate canvas visualization messages.
+%%
+oe:result_set_show(QueryID,ResultSet) :-
+	result_set_has_object(ResultSet),
+	result_set_objects(ResultSet,Objs),
+	maplist(has_base_link_name, Objs, Links),
```

Comment:
this will not work for e.g. facings in REFILLS.
I sent you a mapper function you can use via Jabber, remember?
Please use the one I sent to you